### DISCUSSION TITLE
How to create another admin/special user in the system?

### DISCUSSION THREAD

**Discussion by ziaatrix** [serenity-is/Serenity#7150] [General]
Hi,

By default there is admin user in serenity, where this user is a special user that is able to see all records across all tenants and its working fine.

I need to create another user same exactly as admin. But there is no configuration here to set it as like special user. How to create another admin/special user in the system?

![image](https://github.com/serenity-is/Serenity/assets/118987818/4bc70347-c1e5-44b1-a1cb-a6191d2d5158)

**Comment by ziaatrix** [comment]
@volkanceylan Any idea please?

**Reply by VictorTomaili** [reply]
Please don't mention us directly. 
We didn't have support for multiple admin like this. The super user is designed for one user only. 

You can create a role for multitenant view and bypass tenant filter for this role. The tenant filter is in your code base, and you can add any business to there.

**Comment by LSNicholls** [comment]
@ziaatrix beyond the specialized multi-tenant logic, you will also have to search for the strings 'admin' and "admin" (with the delimiters; the former will be in script and the latter will be in CS), and evaluate whether you need to change the logic where you find these hardcoded strings to allow your additional user(s) to have the same abilities as admin is being given.
